docs/source/write_first_app.rst

```diff
@@ -512,100 +880,42 @@ the application, smart contract, peers and ordering service work together to
 keep the ledger consistent across the network is called consensus, and it is
 explained in detail in this `section <./peers/peers.html>`_.
 
-To see that this transaction has been written to the ledger, go back to
-``query.js`` and change the argument from ``CAR4`` to ``CAR12``.
-
-In other words, change this:
-
-.. code:: bash
-
-  const result = await contract.evaluateTransaction('queryCar', 'CAR4');
-
-To this:
-
-.. code:: bash
-
-  const result = await contract.evaluateTransaction('queryCar', 'CAR12');
-
-Save once again, then query:
-
-.. code:: bash
-
-  node query.js
-
-Which should return this:
-
-.. code:: bash
-
-  Wallet path: ...fabric-samples/fabcar/javascript/wallet
-  Transaction has been evaluated, result is:
-  {"color":"Black","docType":"car","make":"Honda","model":"Accord","owner":"Tom"}
-
-Congratulations. You’ve created a car and verified that its recorded on the
-ledger!
-
-So now that we’ve done that, let’s say that Tom is feeling generous and he
-wants to give his Honda Accord to someone named Dave.
 
-To do this, go back to ``invoke.js`` and change the smart contract transaction
-from ``createCar`` to ``changeCarOwner`` with a corresponding change in input
-arguments:
-
-.. code:: bash
-
-  await contract.submitTransaction('changeCarOwner', 'CAR12', 'Dave');
-
-The first argument --- ``CAR12`` --- identifies the car that will be changing
-owners. The second argument --- ``Dave`` --- defines the new owner of the car.
-
-Save and execute the program again:
-
-.. code:: bash
-
-  node invoke.js
-
-Now let’s query the ledger again and ensure that Dave is now associated with the
-``CAR12`` key:
-
-.. code:: bash
-
-  node query.js
-
-It should return this result:
-
-.. code:: bash
-
-   Wallet path: ...fabric-samples/fabcar/javascript/wallet
-   Transaction has been evaluated, result is:
-   {"color":"Black","docType":"car","make":"Honda","model":"Accord","owner":"Dave"}
+The ``evaluateTransaction`` method represents one of the simplest interactions
```

Comment:
if possible, move this para about evaluateTransaction before the description about `submitTransaction` in line 859, to describe the simpler evaluateTransaction first as a natural progression

Reply:
Agree, it flows better this way.  Done.